src/app/api/admin/categories/route.ts

```
@@ -0,0 +1,99 @@
+    req,
+    withPermission(
+      req,
+      'category' as ResourceType,
+      'create' as Permission,
+      async (validatedReq) => {
+        try {
+          // Get tenant context
+          const tenantId = validatedReq.headers.get('x-tenant-id') as string;
+
+          // Get tenant context
+          const tenantId = validatedReq.headers.get('x-tenant-id') as string;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'tenantId'. Consider to delete it or rename it.

'tenantId' is defined here:


(lint/suspicious/noRedeclare)